apps/frontend/src/pages/project/libs/components/project-details-menu/project-details-menu.tsx

```diff
@@ -1,50 +1,67 @@
 import { Menu, MenuItem } from "~/libs/components/components.js";
 import { AppRoute, PermissionKey } from "~/libs/enums/enums.js";
 import { checkHasPermission, configureString } from "~/libs/helpers/helpers.js";
-import { useAppSelector, useCallback, usePopover } from "~/libs/hooks/hooks.js";
+import { useCallback, usePopover } from "~/libs/hooks/hooks.js";
+import { type PermissionGetAllItemResponseDto } from "~/modules/permissions/permissions.js";
 
 type Properties = {
 	onEdit: () => void;
 	projectId: number;
+	userPermissions?: PermissionGetAllItemResponseDto[];
```

Comment:
It shouldn't be optional, we always will pass at least empty array here
```suggestion
	userPermissions: PermissionGetAllItemResponseDto[];
```